.travis.yml

```diff
@@ -26,16 +26,16 @@ script:
   - "make depend"
   - "make build"
   - "make test"
-  - "make image"
   - "make check-go-format"
```

Comment:
side note: why do we check the format after building? wouldn't it be faster to do it before instead?

Reply:
Good point. Previously we checked format after building an image (!) so I moved it one step up but yeah.. checking format should be our first think to do :)

Reply:
Done in 6b78ede71dfbcb3a9769946259b2998045888490